pages/common/dlv.md

```diff
@@ -0,0 +1,24 @@
+# dlv
+
+> Debugger for the Go programming language.
+> More information: <https://github.com/go-delve/delve/blob/master/Documentation/usage/dlv.md/>.
+
+- Compile and begin debugging main package in current directory, or the package specified:
```

Comment:
> or the package specified

This should be put in a separate example.

Reply:
@sbrl Thanks for your detailed notes! I have made the changes and separated this section into two examples.